src/scripting/lua_map.cc

```diff
@@ -3343,26 +3324,11 @@ TrainingSiteDescription
 */
 const char LuaTrainingSiteDescription::className[] = "TrainingSiteDescription";
 const MethodType<LuaTrainingSiteDescription> LuaTrainingSiteDescription::Methods[] = {
+   METHOD(LuaTrainingSiteDescription, trained_soldiers),
    {nullptr, nullptr},
 };
 const PropertyType<LuaTrainingSiteDescription> LuaTrainingSiteDescription::Properties[] = {
-   PROP_RO(LuaTrainingSiteDescription, food_attack),
-   PROP_RO(LuaTrainingSiteDescription, food_defense),
-   PROP_RO(LuaTrainingSiteDescription, food_evade),
-   PROP_RO(LuaTrainingSiteDescription, food_health),
-   PROP_RO(LuaTrainingSiteDescription, max_attack),
-   PROP_RO(LuaTrainingSiteDescription, max_defense),
-   PROP_RO(LuaTrainingSiteDescription, max_evade),
-   PROP_RO(LuaTrainingSiteDescription, max_health),
    PROP_RO(LuaTrainingSiteDescription, max_number_of_soldiers),
-   PROP_RO(LuaTrainingSiteDescription, min_attack),
-   PROP_RO(LuaTrainingSiteDescription, min_defense),
-   PROP_RO(LuaTrainingSiteDescription, min_evade),
-   PROP_RO(LuaTrainingSiteDescription, min_health),
-   PROP_RO(LuaTrainingSiteDescription, weapons_attack),
-   PROP_RO(LuaTrainingSiteDescription, weapons_defense),
-   PROP_RO(LuaTrainingSiteDescription, weapons_evade),
-   PROP_RO(LuaTrainingSiteDescription, weapons_health),
    {nullptr, nullptr, nullptr},
 };
```

Comment:
Please don't remove Lua properties or methods for API compatibility. You can mark them as "deprecated since 1.2", but if they can at all be kept around they should be.

Reply:
I have checked carefully whether they are still in use in our lua code. (i did a fullsearch over the data directory.  It would be possible to keep min and max values as they are still availablein c++ code of trainingsite. keeping the food and weapons values would mean either keeeping the unnecessary definitions in the lua files from where they were parsed in the trainingsite code or try to rebuild them by summarising elements of the program code, Both is not desirable imho.

Reply:
I reinstated the min_max values is this ok for @Noordfrees